usaspending_api/search/delta_models/award_search_testing.py

```diff
@@ -0,0 +1,96 @@
+award_search_testing_sql_string = r"""
```

Comment:
Any reason not to reuse the `award_search_create_sql_string`? Same with `transaction_search` below.

Reply:
removed

Reply:
This still exists for `usaspending_api/transactions/delta_models/transaction_search_testing.py`. Can you remove it as well?